.travis.yml

```diff
@@ -2,8 +2,8 @@ language: ruby
 rvm:
   - 2.3.0
 
-services:
-  - postgresql
+addons:
+  postgresql: "9.4"
```

Comment:
Right - I'm just thinking that 9.5 is out, and 9.6 is in beta and may be out soon, and we'll want to verify so our tests don't drift from production where we can avoid that.